plugins/inventory/nb_inventory.py

```diff
@@ -264,6 +264,7 @@ def _fetch_information(self, url):
                     headers=self.headers,
                     timeout=self.timeout,
                     validate_certs=self.validate_certs,
+                    follow_redirect=True,
```

Comment:
Do you mind changing this to an option?

1. Update documentation to add `follow_redirects` option
2. Gather option within the `parse` function that we grab all the possible options
3. Change L267 to use `self.follow_redirects`

Reply:
Just implemented it, keeping `urllib2` as the default value to avoid introducing a breaking change.